.github/workflows/native-posix-tests.yml

```diff
@@ -0,0 +1,53 @@
+name: Native Posix tests
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches:
+      - main
+      - 'v*-branch'
+  push:
+    branches:
+      - main
+      - 'v*-branch'
+    tags:
+      - v*
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  native_posix_test:
+    runs-on: ubuntu-latest
+    name: Run native_posix tests and generate coverage report
+
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v2
+        with:
+          path: ncs/nrf
+          fetch-depth: 0
+
+      - name: West init and update
+        uses: docker://nordicplayground/nrfconnect-sdk:main
```

Comment:
how does this work? I only saw docker being used with "container" keyword https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container

Reply:
It does work, but it launches a new container for each step. You should use `container:` here unless there is a problem with that.

Reply:
There was a problem with doing that. But I cant remember what It was. I will give it a go. Thanks!